### PR TITLE
RPM based appliance build

### PR DESCRIPTION
**Pull request by simaishi**

### CODE REVIEW

File: kickstarts/base.ks.erb [hunk 1]
```diff
@@ -69,30 +69,18 @@ unset DEBUG
...
-<%= render_partial "post/bundler" %>
-
-# needs to run *after* post/bundler
-<%= render_partial "post/ui_compile" %>
-
-<%= render_partial "post/rake_tasks" %>
-
-# appliance_root="/opt/manageiq/manageiq-appliance" -- in post/source_setup partial
-$appliance_root/manageiq-setup.sh
+/opt/manageiq/manageiq-appliance/manageiq-setup.sh
```
Comment (Fryguy) [inline kickstarts/base.ks.erb:83]: Does this needs to be productizable? (We can do it in a follow up PR)

Comment (simaishi) [reply kickstarts/base.ks.erb]: Updated.

File: kickstarts/partials/main/repos.ks.erb [hunk 2]
```diff
@@ -12,3 +12,4 @@ repo --name=google-cloud-compute --baseurl=https://packages.cloud.google.com/yum
 # Please also add to "post install repos" post/repos partial
 repo --name=ansible-runner --baseurl=https://releases.ansible.com/ansible-runner/rpm/epel-8-x86_64/
 repo --name=manageiq-ManageIQ-Master --baseurl=https://copr-be.cloud.fedoraproject.org/results/manageiq/ManageIQ-Master/epel-8-x86_64/
+repo --name=simaishi-test --baseurl=https://copr-be.cloud.fedoraproject.org/results/simaishi/test/epel-8-x86_64/
```
Comment (chessbyte) [inline kickstarts/partials/main/repos.ks.erb:15]: should `simaishi-test` be a more generic name?

**Comment by chessbyte** [comment]
Love the +9 -270 change count on this PR!

File: kickstarts/partials/main/repos.ks.erb (hunk 2 above)
Comment (simaishi) [reply kickstarts/partials/main/repos.ks.erb:15]: This is temporary for testing purpose. https://github.com/ManageIQ/manageiq-appliance-build/pull/415/commits/a40faff8dcb961cd7b9048f8ffca835a6b318580 will be removed before merge.

Comment (simaishi) [reply kickstarts/partials/main/repos.ks.erb]: Removed the commit.

**Comment by simaishi** [comment]
Jansa backport details (replaced yum repo with 10-jansa):

```
$ git log -1
commit f1b75254a010438343f0302156da177ca934f7a3
Author: Jason Frey <fryguy9@gmail.com>
Date:   Thu May 28 11:35:05 2020 -0400

    Merge pull request #415 from simaishi/rpm_build_kickstart

    RPM based appliance build

    (cherry picked from commit 7077163a4df8e0538f372e27c65a8b1541ad739a)
```

**Pull request merged**